src/orquestra/sdk/_base/cli/_dorq/_entry.py

```diff
@@ -396,24 +400,41 @@ def status():
     status,
 )
 
+
+# region: login
+class GroupWithDefaultCommand(cloup.Group, DefaultGroup):
+    ...
+
+    def get_help(self, ctx) -> str:
+        # Hack to get the help for `orq login` to make sense
+        sub = super().get_help(ctx).split("\n")
+        return auth.get_help(ctx) + "\n\n" + "\n".join(sub[sub.index("Commands:") :])
+
+
+@dorq.group(cls=GroupWithDefaultCommand, default="auth", invoke_without_command=False)
+def login():
+    """Commands related to logging in to clusters."""
+    pass
```

Comment:
I think this works!

Reply:
I certainly thought so when I committed it! XD

Reply:
this was in reply to:

> I'm unhappy about the usage of click_default_group to achieve this since it a) adds an additional 3rd party dependency for us, and b) lacks stubs for typing. I might rework this to avoid using it, but I want to give everyone a chance to discuss the CLI structure before I commit to spending more time making this precise vision of it work.

Reply:
Ah, that makes more sense!